### DISCUSSION TITLE
2024/12/30/gmod-git-setup

### DISCUSSION THREAD

**Discussion by giscus** [AMD-NICK/blog.amd-nick.me#39] [Blog comments]
# 2024/12/30/gmod-git-setup

Я годами не мог придумать, как сделать, чтобы на Git хранились только самые необходимые файлы и папки без мусорных (cache, backgrounds, console.log и т.д.). Казалось бы, простая задача, но решил я ее лишь недавно.

https://blog.amd-nick.me/2024/12/30/gmod-git-setup

**Comment by DEXTER251** [comment]
Пример .gitignore в моем случае
Штука довольно индивидуальна, как например беру только свой гейммод в учет
```
.vscode
*.mdmp
*.vmt
*.vtf
*.mdl
*.vvd
*.phy
*.wav
*.vtx
*.mp3
*.png
*.bsp
*.moon
*.zip
*.dll
steamapps
cstrike
platform
bin
sourceengine
steam_cache
srcds.exe
steam_appid.txt
ThirdPartyLegalNotices.txt

garrysmod/*
!garrysmod/addons/
!garrysmod/lua/

garrysmod/gamemodes/*
!garrysmod/gamemodes/darkrp/
```
